src/test/graphql_test.erl

```diff
@@ -327,6 +327,15 @@ variable_enum_two_test() ->
     {<<"arg">>, [{<<"enum">>, <<"TWO">>}]}
   ], errors => []}, graphql:execute(graphql_test_schema:schema_root(), Document, VariableValues, #{}, #{})).
 
+variable_enum_three_test() ->
+  Document = <<"query($var: EnumOneTwo!) { arg(enum: $var) { enum } }">>,
```

Comment:
Incorrect query.

Field arg accept nullable enum, but argument passed in incorrect type. You cannot pass non null value for nullable type.

You can create you own argument in field <<"arg_non_null">> like
<<"enum">> => #{
  type => ?NON_NULL(fun enumOneTwo/0)
}

In this case all may work